lib/protobuf/field/bytes_field.rb

```diff
@@ -58,11 +58,12 @@ def define_setter
         message_class.class_eval do
           define_method(method_name) do |val|
             begin
-              val = "#{val}" if val.is_a?(Symbol)
-
-              if val.nil?
+              case val
+              when String, Symbol then
```

Comment:
style q: why use `then` for a multiline case? I'm only used to using it for single-line